src/main/java/org/broadinstitute/hellbender/tools/spark/sv/SVIntervalTree.java

```diff
@@ -0,0 +1,904 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
```

Comment:
(there's actually 2 interval trees in htsjdk, one of which you wrote, as well as `OverlapDetector` which is built on your interval tree)